lib/rubocop/cop/rails/application_record.rb

```diff
@@ -28,6 +28,12 @@ class ApplicationRecord < Cop
         # rubocop:disable Layout/ClassStructure
         include RuboCop::Cop::EnforceSuperclass
         # rubocop:enable Layout/ClassStructure
+
+        def autocorrect(node)
+          lambda do |corrector|
+            corrector.replace(node.source_range, self.class::SUPERCLASS)
+          end
+        end
```

Comment:
Moved the Rails `application_record` and `application_job` autocorrections directly into the cop because they were so slim.